README.md

```diff
@@ -1,10 +1,12 @@
-# Creative Commons Licenses
+# cc-legal-tools-app-app
```

Comment:
The word "app" is in there twice and it might be good to format the text for humans

```suggestion
# Creative Commons Legal Tools
```